UtopiaCity/Data/Migrations/AirportMigrations/20210808153541_Flight_Last.cs

```diff
@@ -6,9 +6,9 @@ public partial class Flight_Last : Migration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
         {
-            migrationBuilder.DropColumn(
-                name: "TravelDistance",
-                table: "Flights");
+//            migrationBuilder.DropColumn(
```

Comment:
?

Reply:
There were problems in time of using the "update-database" command so I commented out it